app/controllers/concerns/shopify_app/authenticated.rb

```diff
@@ -9,7 +9,7 @@ module Authenticated
       include ShopifyApp::LoginProtection
       include ShopifyApp::EmbeddedApp
       before_action :login_again_if_different_user_or_shop
-      around_action :shopify_session
+      around_action :activate_shopify_session
```

Comment:
I see what you are doing here however, people write their own integrations with this action. People wrap certain parts of their code in shopify_session. And since you are not removing the method named this, this could cause some serious confusion

Reply:
You're right, so I'll change `shopify_session` to `current_shopify_session` so that it's a more obvious breaking change.